src/styles/sidebar/components/top-bar.scss

```diff
@@ -1,10 +1,12 @@
 @import '../../base';
 @import '../../mixins/responsive';
+@import '../../mixins/buttons';
 
 .top-bar {
   background: $white;
-  border-bottom: solid 1px $gray-lighter;
+  border-bottom: solid 1px $grey-3;
   height: $top-bar-height;
+  // TODO bring into standard rhythm of typography
```

Comment:
This really bugs me :). There is no precedent for `15px` fonts in the client, so I presume this was done to make stuff "fit" into the `TopBar` at some point in the distant past. I felt the need to add a TODO because I'd like to see us fix this, however, changing it now disrupts things too much.

Reply:
roger that